### PR TITLE
JEN-967

### PR DESCRIPTION
**Pull request by abychko**
8.0 PS sanitizer build did not even finish compiling
[+] choose docker-32gb instead of docker if all three -DWITH_ASAN=ON -DWITH_ASAN_SCOPE=ON -DWITH_UBSAN=ON are given

### CODE REVIEW

**Comment by laurynas-biveinis** [comment]
Please show testing

**Comment by abychko** [comment]
test run started with 'docker'. variable is set to 'docker-32' https://ps2.cd.percona.com/view/8.0/job/test-percona-server-8.0-pipeline/9/console

**Comment by laurynas-biveinis** [comment]
That is wrong UI then: we need the default in the param window changed, not the choice override when the build starts

**Comment by abychko** [comment]
there is no javascript possibility to change the UI combobox from docker to docker-32 is some other params are chosen

File: jenkins/pipeline.groovy [hunk 1]
```diff
@@ -127,6 +133,7 @@ pipeline {
                 }
 
                 sh 'echo Prepare: \$(date -u "+%s")'
+                echo "Running on ${LABEL} label JEN-967"
```
Comment (laurynas-biveinis) [inline jenkins/pipeline.groovy]: Please remove before merging

**Comment by laurynas-biveinis** [comment]
OK, then please adjust the instance size comment to include "will be set to -32gb if -DWITH_ASAN... are selected"

**Comment by abychko** [comment]
Label will be adjusted to docker-32gb due to DWITH_ASAN
https://ps2.cd.percona.com/view/8.0/job/test-percona-server-8.0-pipeline/12/console

**Comment by laurynas-biveinis** [comment]
Please write out all three options "-DWITH_ASAN=ON -DWITH_ASAN_SCOPE=ON -DWITH_UBSAN=ON" in the comment

**Comment by abychko** [comment]
extended comment

**Comment by laurynas-biveinis** [comment]
No I mean the comment in the param window, where you can choose between docker and docker-32gb

**Comment by abychko** [comment]
please give me full string to replace 'Run build on specified instance type'

**Comment by laurynas-biveinis** [comment]
Run build on specified instance type. If "-DWITH_ASAN=ON -DWITH_ASAN_SCOPE=ON -DWITH_UBSAN=ON" is specified above, docker-32gb is used unconditionally.

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**